connector/avro/src/main/scala/org/apache/spark/sql/avro/AvroUtils.scala

```diff
@@ -43,6 +43,9 @@ import org.apache.spark.sql.types._
 import org.apache.spark.util.Utils
 
 private[sql] object AvroUtils extends Logging {
+
+  AvroFileFormat.registerCustomAvroTypes()
```

Comment:
Could you provide a test coverage for this line? The test case on this PR passes without this.